app/digests/exceptions.py

```diff
@@ -0,0 +1,28 @@
+import logging
+from typing import List
+
+from django.conf import settings
+from rest_framework.response import Response
+
+LOGGER = logging.getLogger(__name__)
+
+
+def format_error_path(path: List[str]) -> str:
+    return '.'.join([str(item) for item in path])
+
+
+def generate_error_string(message: str, path: List[str]) -> str:
+    equals_str = ''
+    if len(path):
+        equals_str = ' = '
+
+    return '{0}{1}{2}'.format(format_error_path(path), equals_str, message)
+
+
+def validation_error_handler(exception: Exception) -> Response:
+    LOGGER.exception(exception)
```

Comment:
IIRC `Logger.exception` already has all the information it needs about the exception from global state (like the stack trace), so the `message` argument here should include the context instead; we don't have much else to log but perhaps a fixed explanatory message?

Reply:
Now passing in the generated error message to be logged in addition to the normal stack trace. 👍